blog/2023-07-27-product-localization.md

```diff
@@ -0,0 +1,168 @@
+
+### Tools and processes
+
+**Selecting the appropriate tools and processes** for your localization project is crucial based on the localization strategy you have defined in the previous steps.
+
+Ensure that the **tools** you opt for align seamlessly with your localization strategy. For instance, if Machine Translation (MT) is necessary for your project, choosing a Translation Management System (TMS) that fully supports this feature is critical. Additionally, if your aim is to assess the effectiveness of your localization efforts, devise a method to gather and analyze relevant data.
+
+Likewise, the **processes** you implement should complement your localization strategy. When incorporating MT, consider creating a process for post-editing the machine-translated content within the TMS to ensure optimal results. If you’re using a TMS, you can also create a process for managing the translation workflow, from translation to reviewing and approving the translated content.
+
+## Localization strategy examples
+
+### McDonald’s
+
+Even though McDonald’s is a global brand and is considered a globalization behemoth, they still localize their products to fit the local market. In India, McDonald’s replaced the two traditional beef patties in a Big Mac with chicken ones to create the [Maharaja Mac](https://mcdonaldsblog.in/2016/11/chicken-maharaja-mac-the-indian-answer-to-big-mac/). The reason for this change is that in India, cows are considered sacred animals, and in most states of India, there is a law prohibiting beef consumption.
+
+In the Philippines, McDonald’s has introduced [McSpaghetti](https://www.mcdonalds.com.ph/our-food/chicken-platters) to their menu. The dish combines spaghetti noodles, tomato sauce, and grated cheese. This change is because spaghetti is a popular dish in the Philippines and is often served during birthday parties.
+
+### Uber
+
+Uber is an excellent example of a company offering genuinely local services. In 2015, Uber launched [UberAuto](https://www.uber.com/in/en/ride/uber-auto), which allows users to book an auto-rickshaw in India. As cash is a common payment method in India, Uber has also decided to allow users to pay for their rides in cash.
+
+### Slack
+
+Slack has localized its product into more than 10 languages while maintaining a consistent brand identity and tone of voice. In their blog post, [How we localize Slack](https://slack.com/blog/collaboration/how-we-localize-slack), they show examples of how they localized (not just translated) messages and icons in their product. For example, the message displayed when you’ve read everything in your “Unreads” section, ”You’re all caught up. Here’s a pony.”, does not make sense in Spanish. So in Spanish, they localized it as “Has leído todo. Aquí tienes un caramelo de regalo.”, meaning “You’ve read everything. Here’s a candy gift.” – the icon was also changed to a candy.
+
+## Conclusion

```

Comment:
I'd add some different headline, than just consluion. e.g. Conclusion: The Crucial Role of Product Localization in Global Expansion